src/components/Panel/TicketDraggableContext.ts

```diff
@@ -1,9 +0,0 @@
-import { createContext } from "react";
-
-export interface TicketDraggable {
-  draggable: boolean;
-}
-
-export default createContext<TicketDraggable>({
-  draggable: false,
-});
```

Comment:
I'm sorry this was a brilliant solution, but I think it was just adding complexity just to pass single property to a component, so I couldn't find any value in setting this property in context...